src/providers/test-result/test-result.ts

```diff
@@ -120,4 +126,27 @@ export class TestResultProvider {
 
     return of(ActivityCodes.PASS);
   }
+
+  private calculateCatCDndSubCategoryTestResult = (
```

Comment:
should be named calculateCatDAndSubCategoryTestResult

Reply:
fixed